specification/cognitiveservices/data-plane/CustomImageSearch/stable/v1.0/CustomImageSearch.json

```diff
@@ -104,9 +104,7 @@
             "in": "query",
             "description": "The identifier for the custom search configuration",
             "required": true,
-            "type": "integer",
-            "format": "int64",
-            "minimum": 0
+            "type": "string"
```

Comment:
Please also update the swagger "title" to... "Custom Image Search Client"
Long time back we had an discussion with Ronak, all cognitive services SDKs should be named as "***Client"
And "Name changes will happen to Bing swaggers next time we update Bing API swaggers"